src/EditorFeatures/Core/LanguageServer/AbstractInProcLanguageClient.cs

```diff
@@ -267,6 +268,13 @@ public Task OnServerInitializedAsync()
     /// </summary>
     public Task AttachForCustomMessageAsync(JsonRpc rpc) => Task.CompletedTask;
 
+    private static PropertyCollection CreateStjPropertyCollection()
+    {
+        var collection = new PropertyCollection();
+        collection.AddProperty("lsp-serialization", "stj");
```

Comment:
are these read elsewhere?  consider constants.

Reply:
not by us, but it is ready by the LSP client.  will switch to constants

Reply:
if the client reads this, can you add a comment to that effect?

Reply:
(i'm actually ok with no constants in this case since we don't use these values anywhere else).  but we should doc at least what's going on :)